openhtf/util/logs.py

```diff
@@ -88,6 +89,10 @@ def MyPhase(test, helper):
     '--verbosity', default=DEFAULT_LEVEL, choices=LEVEL_CHOICES,
     action=argv.StoreInModule, target='%s.DEFAULT_LEVEL' % __name__,
     help='Console log verbosity level (stderr).')
+ARG_PARSER.add_argument(
+    '--test-record-verbosity', default=DEFAULT_RECORD_VERBOSITY, choices=LEVEL_CHOICES,
+    action=argv.StoreInModule, target='%s.DEFAULT_RECORD_VERBOSITY' % __name__,
+    help='Console log verbosity level for test record (not framework logger) (stderr).')
```

Comment:
The twin parentheticals at the end of the help string might be a bit confusing.

Reply:
Fair point, will edit.